### PR TITLE
ISSUE_TEMPLATE update

### PR DESCRIPTION
**Pull request by mchlwellman**
Copied the bug and issue templates from vateam into this repo.  The branch says api but I used the templates from vateam, as there is one small difference in the api repo (BUG label is not in the bug report).

### CODE REVIEW

**Review by k-macmillan — DISMISSED** [review]

**Review by MackHalliday — DISMISSED** [review]
gif

File: .github/ISSUE_TEMPLATE/story-template.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+- [ ] 
+
+## Acceptance Criteria
+<!--_**Required**_
+- e.g. when I send an email using an identifier, we retrieve the user’s email address in VA Profile and successfully deliver the email to that recipient-->
+
+<!--
+## QA Considerations
+_For QA to populate. Leave blank if QA is not applicable on this ticket._
```
Comment (EvanParish) [inline .github/ISSUE_TEMPLATE/story-template.md:40]:
```suggestion
_Populate with suggested QA actions if applicable. Leave blank if QA is not applicable on this ticket._
```

Could we replace this comment on both templates please?

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<!--_**Required.** Describe the impact this bug has on our system, clients, and/or team._-->
+
+## Expected Behavior
+<!--_**Required.** Describe the desired outcome if this were functioning as expected. Include a checklist if applicable._-->
+
+- [ ] 
+
+<!--
+## QA Considerations
+_For QA to populate. Leave blank if QA is not applicable on this ticket._
```
Comment (EvanParish) [inline .github/ISSUE_TEMPLATE/bug-report-template.md:41]:
```suggestion
_Populate with suggested QA actions if applicable. Leave blank if QA is not applicable on this ticket._
```

Could we replace this comment on both templates please?

**Review by MackHalliday — APPROVED** [review]

**Review by k-macmillan — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report-template.md (hunk 2 above)
Comment (mchlwellman) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Updated

**Pull request merged**